packages/api/src/server.ts

```diff
@@ -39,12 +39,12 @@ export class WepublishServer {
 
     methodsToProxy.forEach(mtp => {
       if (mtp.key in dbAdapter) {
+        const dbAdapterKeyTyped = mtp.key as keyof typeof dbAdapter
```

Comment:
I think this file changes needs more enhancement, as this line isn't satisfying for line 47 for example.

Reply:
Yes the `// @ts-ignore` for line 47 is still there as I could not find a simple way to get rid of that. I will investigate further.

Reply:
Not sure how to get rid of the `// @ts-ignore` for line 47, as this one is a bit beyond my current knowledge of TypeScript. I believe it is not straightforward as `methodName` is only known at runtime and it is used as a deep key of the `dbAdapter` object. @bishoymelek Perhaps you have some ideas?

Reply:
@gianantoniopini: this whole implementation should get refactored anyway so I am already happy if we can get rid of the `// @ts-ignore` you did 😃 .